x-pack/plugin/build.gradle

```diff
@@ -7,6 +7,7 @@ apply plugin: 'elasticsearch.rest-test'
 archivesBaseName = 'x-pack'
 
 dependencies {
+  testCompile project(xpackModule('core'))
```

Comment:
Should we add a comment here that this only exists to satisfy intellij? it should be picked up transitively, as it is when running from the CLI right?

Reply:
Added.